analysis_benchmark.json

```diff
@@ -0,0 +1,5 @@
+{
+  "time": 3.488,
+  "issues": 0,
+  "missingDartDocs": 0
+}
```

Comment:
you probably didn't mean to include this file?

Reply:
deleted